server/index.ts

```diff
@@ -33,14 +33,15 @@ export let openGraphPlugin =
 
       if (!page.frontmatter.layout) {
         head.push(['meta', { property: 'og:type', content: 'article' }])
-
-        head.push([
-          'meta',
-          {
-            content: new Date(page.frontmatter.date!).toISOString(),
-            property: 'article:published_time',
-          },
-        ])
+        if (page.frontmatter.date {
```

Comment:
```suggestion
        if (page.frontmatter.date) {
```

Reply:
Sorry... Pushed fix.  That's what I get for trying to quick fix inside GitHub editor. :(